Branch-SDK/src/main/java/io/branch/referral/Branch.java

```diff
@@ -1827,12 +1826,36 @@ public void setIdentity(@NonNull String userId, @Nullable BranchReferralInitList
         }
     }
 
+    /**
+     * Gets all available cross platform ids.
+     *
+     * @param callback An instance of {@link io.branch.referral.util.BranchLastAttributedTouchData} to callback with cross platform ids
+     *
+     */
     public void getCrossPlatformIds(BranchCrossPlatformIdListener callback) {
         new BranchCrossPlatformId(callback, context_);
     }
 
+    /**
+     * Gets the available last attributed touch data.
+     *
+     * @param callback An instance of {@link io.branch.referral.util.BranchLastAttributedTouchData} to callback with last attributed touch data
+     *
+     */
+    public void getLastAttributedTouchData(BranchLastAttributedTouchDataListener callback) {
+        prefHelper_.setLATDAttributionWindow(null);
+        new BranchLastAttributedTouchData(callback, context_);
```

Comment:
This is making a network request in the constructor.

Reply:
Fixed.